### DISCUSSION TITLE
WARNING  Will not infuse unfinished block

### DISCUSSION THREAD

**Discussion by JULINJAN** [Chia-Network/chia-blockchain#5359] [Support]
WARNING  Will not infuse unfinished block b22d43495739556337ffc58fc647e9ae4cb7b8517c78fc363fc15ff9736df839 sp total iters 961419952128, because there is another infusion before its SP

I have this warning on my log
It seems like criticle
my chia version is 1.1.5 
Thanks for help
love chia

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
